activation/activation_test.go

```diff
@@ -353,3 +353,52 @@ func TestBuilder_PublishActivationTxSerialize(t *testing.T) {
 	bt2, err := types.AtxAsBytes(a)
 	assert.Equal(t, bt, bt2)
 }
+
+func TestBuilder_PublishActivationTx_PosAtxOnSameLayerAsPrevAtx(t *testing.T) {
```

Comment:
doesn't line 242 test that?

Reply:
Not sure what you mean... 

Line 242 checks that you can't publish two ATXs in the same epoch.

This test, which currently fails, checks that we select a positioning ATX that's on the same layer ID as the previous ATX, so that the layer ID relative to the epoch stays constant. This can't currently pass, since we select a positioning ATX randomly from the current epoch without looking at the specific layer ID.